resources/js/components/menu-images.tsx

```diff
@@ -38,6 +38,11 @@ export default class MenuImages extends React.Component<Props> {
     return Math.min(0, this.index);
   }
 
+  @computed
+  private get images() {
+    return shuffle(this.props.images);
```

Comment:
Forgot to mention `props` used by `@computed`s or any other reaction should be copied locally due to changes in newer mobx-react and React (or just initial the field as shuffled? 🤔 )